src/commands/RunCommand.js

```diff
@@ -64,8 +64,7 @@ export default class RunCommand extends Command {
     }
 
     if (!this.packagesWithScript.length) {
-      callback(new Error(`No packages found with the npm script '${this.script}'`));
-      return;
+      this.logger.warn(`No packages found with the npm script '${this.script}'`);
     }
```

Comment:
How about we just exit early here, instead of proceeding to the `execute()` phase after batching an empty array?
```
return callback(null, false);
```